src/components/ArticleCard.jsx

```diff
@@ -1,10 +1,14 @@
 function ArticleCard({ article }) {
-  
-  if(article){
+
+    
+    if(article){
+      const createdAt = new Date (article.created_at)
   return (
     <div className="articlecard"> 
     <img className="articlelistimg" src={article.article_img_url}></img>
       <h3> {article.title} Article Card</h3>
+      <div>Votes {article.votes}</div>
```

Comment:
at the moment your votes are displaying above the article image - we should group the similar info together

Reply:
if this is because you have plans to style this in a particular way then ignore this comment

Reply:
Decided I wanted vote count to appear in the top right of the article card and this will be updated with styling in later stages.